_includes/objectives.html

```diff
@@ -0,0 +1,13 @@
+<section class="flex flex-row md:gap-5 p-2 md:p-10 w-full bg-gray-300 object-cover text-black">
+  <div class="w-full md:w-1/2 p-10 relative p-2">
+    <button class="bg-black rounded-[4px] pt-2 pr-3 pb-2 pl-3 text-lg text-white px-6 py-3 font-medium font-sans mb-8">
```

Comment:
This does not have to be button, since it has not an associated action, and it might confuse users. Change it to a div.